build.env

```diff
@@ -23,7 +23,7 @@ GO111MODULE=on
 COMMITLINT_VERSION=latest
 
 # static checks and linters
-GOLANGCI_VERSION=v1.39.0
+GOLANGCI_VERSION=v1.42.0
```

Comment:
please revert this change. we are going to get a lot of linter errors for this one.

Reply:
yeah, done. I added it to check the CI here picks the errors I get locally, reverted this commit